### PR TITLE
Weekly update for the 2023-11-21.

### PR DESCRIPTION
**Pull request by cube2222**

### CODE REVIEW

File: WEEKLY_UPDATES.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+    - We're productionizing the code, making sure it's stable, and will start hydrating the registry soon.
+  - As usual, a bunch of various bug-fixes.
+  - The installation docs have been drastically improved, and we're in the process of writing docs for the testing feature.
+- How can I help?
+  - Since the alpha release is out, right now the best way to help is to take the alpha for a test drive and see if there are any bugs / issues.
+  - Other than that, the best way to help is to create issues, discuss on issues, and spread the word about OpenTofu.
+  - There are some occasional minor issues which are accepted and open to external contribution, esp. ones outside the release-blocking path. We’re also happy to accept any minor refactors or linter fixes. [Please see the contributing guide for more details](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md).
+    - You can look for a label, `good-to-pick-up`, to find issues we've deemed are the best to pick up for external contributors.
```
Comment (janosdebugs) [inline WEEKLY_UPDATES.md:22]: I don't think we have any issues labeled with this, this should be written in the future tense.

**Review by janosdebugs — APPROVED** [review]
Looks good, minor text change may be needed.

File: WEEKLY_UPDATES.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+## 2023-11-21
+
+Hello!
+
+Another week another update! This week has been very steady. We've been moving forward with the stable registry, and there's been no surprises so far.
+
+Additionally, a new engineer has joined the OpenTofu Core Team and will be 100% dedicated to OpenTofu development. Welcome, @janosdebugs!
+
+- Current Status and Up Next
+  - Registry
```
Comment (Yantrio) [inline WEEKLY_UPDATES.md:12]: It may be nice to link to the RFCs that we're using for implementation details.

File: WEEKLY_UPDATES.md (hunk 1 above)
Comment (cube2222) [reply WEEKLY_UPDATES.md:22]: Well, we have some closed ones! Could be in past tense too :P

Comment (cube2222) [reply WEEKLY_UPDATES.md]: adjusted the message

File: WEEKLY_UPDATES.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
 # Weekly Updates
 
+## 2023-11-21
+
+Hello!
+
+Another week another update! This week has been very steady. We've been moving forward with the stable registry, and there's been no surprises so far.
+
+Additionally, a new engineer has joined the OpenTofu Core Team and will be 100% dedicated to OpenTofu development. Welcome, @janosdebugs!
```
Comment (kislerdm) [inline WEEKLY_UPDATES.md:9]: 🎉

**Review by kislerdm — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**